st/CMSIS_STM32WBAxx/Device/ST/STM32WBAxx/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+---

```

Comment:
I don't think we should check in this .github folder, right?

Reply:
Totally right.

Reply:
But the files are still here :)